packages/core/README.md

```diff
@@ -1,340 +1,67 @@
-# NPM - React Native Embrace
+# React Native Embrace
 
-Embrace gathers the information needed to identify issues and measure performance automatically upon integration. The following React Native guide provides simple instruction on how to call the relevant functions so teams can be provided much needed additional context (logs and user info) and measure the timing of key areas of their app explicitly (moments).
+Embrace gathers the information needed to identify issues and measure performance automatically upon integration.
+The following React Native guide provides simple instruction on how to call the relevant functions so teams can be provided
+much needed additional context (logs and user info) and measure the timing of key areas of their app explicitly (spans).
 
-For additional info please refer to the [React Native Guide](https://docs.embrace.io/docs/quick-quide).
+For additional info please refer to the [React Native Guide](https://docs.embrace.io/docs/react-native).
 
 # Requirements
 
 Only an Embrace App ID and an Embrace API Token.
 
-_If you need an App ID and API Token, contact us at support@embrace.io or on Slack._
+_If you need an App ID and API Token, [Go to our dashboard](https://dash.embrace.io/signup/) to create an account._
 
 # Integration
 
-### Step 1: Add Native Embrace iOS SDK
+## Add the JavaScript library
 
-##### 1.1: Add EmbraceIO's Pod to your Podfile
```

Comment:
to keep this from getting out of sync with our docs I kept just a simplified version of the setup steps and linked to https://docs.embrace.io/docs/react-native for the rest, @TheMutsi let me know if there was any important info here not available on our docs page that should be kept